src/cloud/env/environments.md

```diff
@@ -16,3 +16,13 @@ You can configure application settings, routes, build and deploy actions, and no
 -  [`.magento/services.yaml`]({{ site.baseurl }}/cloud/project/project-conf-files_services.html)—defines the services Magento uses by name and version. For example, this file may include versions of MySQL, PHP extensions, Redis, RabbitMQ, and Elasticsearch. You must open a support ticket to push these changes to Pro plan Staging and Production environments.
 
 When you push code changes, the active environment provisions container updates using the YAML configuration files.
+
+## Update Pro Staging and Production environment configuration
+
+For {{site.data.var.ece}} Pro Staging and Production environments, you can update many configuration options in your local development environment and commit the changes to apply them to these environments. However, you must submit a support ticket to update the following configuration options:
+
+-  Install or update services in the `.magento.services.yaml` file.
+-  Change the configuration for the `mount` and `disk` properties in the `.magento.app.yaml` file.
```

Comment:
section name is `mounts`
Also, I am not sure that it cannot be changed. Found some tickets where it should be changed on Production. Waiting for confirmation from PO.
Need to mention `relationships` section. And, probably `runtime` section (again, waiting confirmation from PO)

Reply:
So I was basing this on an old conversation that I had with Sameer---will share the info with you directly.

Reply:
After talking with PO, having `mounts` editable is like a new feature and we do not use it on our environments yet, so we need to leave it as not editable. I mean change this data through support ticket.
Also, `runtime` section is partially editable, do not have more details.

Reply:
Should we publish these updates, and then create a new PR to add information about the `runtime` section when we have more complete information?

Reply:
sure, thank you